net.devolutions.RDM.yml

```
@@ -0,0 +1,38 @@
+app-id: net.devolutions.RDM
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: remotedesktopmanager
+tags: [proprietary]
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --socket=ssh-auth
+  - --share=network
+  - --filesystem=host
+   
+modules:
+  - name: rdm
+    buildsystem: simple
+    build-commands:
+      - ar x RemoteDesktopManager.deb
+      - tar xf data.tar.xz
+      - mv usr/ /app/
+      - rm -r control.tar.xz data.tar.xz debian-binary
+      - install -D remotedesktopmanager /app/bin/remotedesktopmanager
+      - install -Dm644 net.devolutions.RDM.desktop ${FLATPAK_DEST}/share/applications/net.devolutions.RDM.desktop
+      - install -Dm644 net.devolutions.RDM.png ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/net.devolutions.RDM.png
```

```suggestion
      - install -Dm644 net.devolutions.RDM.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/net.devolutions.RDM.png
```

PNG is NOT scalable.

---

sadly this is a 200x200 PNG file so it's isn't proper either.